### DISCUSSION TITLE
Divider gap

### DISCUSSION THREAD

**Discussion by MarkoGjorgjievski** [saadeghi/daisyui#2058] [Ideas / feature requests]
Hi, 

It might be idea to have a different divider gaps (margins). I often find myself in a need of a different gap than the original (16px).
Probably something like `divider-xs`, `divider-sm` or something similar. What do you guys think?

**Comment by saadeghi** [comment]
You can customize the margin using utility classes like `mx-20` or `my-20` or anything.  
https://play.tailwindcss.com/UxOdk7ps7Z

If it can be modified with a single class name I don't thing using something like `divider-sm` would have an advantage compared to `mx-*` class. It would be just an alias for margin.

Let me know if you have any questions.
